### DISCUSSION TITLE
How to change my pages to a more meaningful URL

### DISCUSSION THREAD

**Discussion by jjwhitmore** [community/community#32154] [Other Feature Feedback, Questions, & Ideas]
Over  the past 10 months or so I built a private github project with set of public github.io pages. The URL is currently of the form my-github-id.github.io/myproject/  . 

Now that I have some experience with the process and the pages are something I am comfortable with, I would like to see if I can change the URL to something of the form...    myproject.github.io/pages/ .  

Does anyone have suggestions?

Thanks.

**Comment by airtower-luna** [comment]
It *may* be possible, depending on some factors:

* The `github.io` domain you get is named like your user *or organization*. You can get a domain for your project if you create an organization for it.
* The part after the domain is the name of the repository, *unless* it's the [user/organization site](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites) (the repository must be named `<user/org name>.github.io`), in which case it's at the root.

If you can't make it work like you want from that, you also have the option to add a custom domain. If you do that your site will deploy at the root of that domain, so you can structure things as you like.
